src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -529,6 +528,14 @@ public GetYourTranscriptsResponse getYourTranscripts(Integer userId) {
         return getYourTranscriptsResponse;
     }
 
+    @Override
+    public List<TranscriberViewSummary> getTranscriberTranscripts(Integer userId, Boolean assigned) {
+        if (assigned != null && TRUE.equals(assigned)) {
```

Comment:
The condition of "not null and true" can be covered by `TRUE.equals(assigned)` only, as it will evaluate to `false` if `assigned` is `null`, so consider removing the redundant `assigned != null &&` check.

Reply:
Done in commit b2783865